### DISCUSSION TITLE
네이버 스크립트 예약이 안되고 있습니다.

### DISCUSSION THREAD

**Discussion by starbw82** [kimytsc/covid-rest-vaccine-macro#87] [Q&A]
예약시도창에서 리스트도 불러오지 못하고있으며,
간혹 불러와도 업데이트는 계속 시도하는데 밑에 볼드 글씨는 업데이트가 안됩니다.

**Comment by kimytsc** [comment]
현재 리스트를 가져오는 네이버 API가 추가된 것을 확인하였습니다.
다만, 기존에 사용하던 API에서도 리스트를 가져오고 있는 것이 확인되었고 신청하는 경로는 아직 변경되지 않았습니다.
그리고 조금전 테스트 결과 아직은 동작하는 것으로 확인되었습니다.

혹시 `DevTools`의 `Network`탭에서 `graphql` 응답이 빨간색으로 노출되고 있는지 확인 가능하신가요?
볼드 글씨가 업데이트 되지 않는다면, `graphql`의 응답이 없다는 의미이며, 일시적인 차단일 가능성이 높습니다.

**Reply by starbw82** [reply]
네 빨간색으로 노출이 되고있습니다.

**Reply by kimytsc** [reply]
빨간색이면 현재 일시적인 차단으로 보입니다.

**Reply by mimkim** [reply]
저도 리스트 못불러와서, 설명 페이지에 링크 걸어주신 예약페이지로 그냥 스크립트 돌렸더니 되긴하더라구요. 예약도 했습니다.
